src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -3533,6 +3533,10 @@ _Default:
                         If declaringSyntax.Parent IsNot Nothing AndAlso TypeOf declaringSyntax.Parent Is TypeBlockSyntax Then
                             Return declaringSyntax.Parent
                         End If
+                    ElseIf TypeOf declaringSyntax Is EnumStatementSyntax Then
```

Comment:
>ElseIf TypeOf declaringSyntax Is EnumStatementSyntax Then [](http://example.com/codeflow?start=20&length=57)

Was the issue specific to Enum types? The issue doesn't mention that.

Reply:
Yes, I identified that after further investigation.